src/test/resources/traffic.json

```diff
@@ -3,20 +3,15 @@
   "match": "traffic",
   "parameters": [
     {
-      "key": "radius_km",
-      "value": "25"
+      "include": "classpath::apikey.json"
```

Comment:
Good idea, but not part of this changeset. There was already no xml include example. Are you OK with making it a pending improvement?